perfkitbenchmarker/providers/gcp/gcp_spanner.py

```diff
@@ -45,8 +45,8 @@ def __init__(self, name, desp, nodes, config, database, ddl):
     self._database = database
     self._ddl = ddl
 
-    # Cloud Spanner is not using the following common flags.
-    self.project = None
+    # Cloud Spanner does not explicitly set the following common flags.
+    self.project = util.GetDefaultProject()
```

Comment:
Added one more flag cloud_spanner_project, which is default to None, and in which case, it will use the default project.